contrib/masterfiles-stage/README.org

```diff
@@ -101,6 +105,15 @@ when developing new staging methods.
     - 88335d36b48c8808b12b48667a463182dc8d0338
     - mytag
 
+- channel_config_file - The path to the config file for multiple policy channels
+  - Example value:
+    - /var/cfengine/policy_channels/channel_to_source.txt
```

Comment:
That's fine, yeah. I was actually thinking of bash associative arrays but those may not be available in older shells:

```
declare -A channels
channels[vcs]="git://..."
...
```

So any implementation is OK with me, as long as it doesn't require separate config files.